tasks/push-rpm-manifests-to-pyxis/tests/mocks.sh

```diff
@@ -44,3 +44,7 @@ function upload_rpm_manifest() {
     rm $LOCK_FILE
   fi
 }
+
+function select-oci-auth() {
```

Comment:
Can you add a check for the calls in at least one of the tests? If you don't want to bother with it now and do it later, that's fine.

Reply:
added in the single arch test